docs/integrations/cicd/octopus-deploy.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Octopus Deploy"
+description: "Learn how to sync secrets from Infisical to Octopus Deploy"
+---
+
+Prerequisites:
+
+- Set up and add secrets to [Infisical Cloud](https://app.infisical.com)
+
+<Steps>
+    <Step title="Create a Service Account for Infisical in Octopus Deploy">
+        Navigate to **Configuration** > **Users** and click on the **Create Service Account** button.
+
+        ![integrations octopus deploy
+        users](/images/integrations/octopus-deploy/integrations-octopus-deploy-user-settings.png)
+
+        Fill out the required fields and click on the **Save** button.
+        ![integrations octopus deploy service
+        account](/images/integrations/octopus-deploy/integrations-octopus-deploy-create-service-account.png)
+    </Step>
+    <Step title="Generate an API Key for your Service Account">
+        On the **Service Account** user page, expand the **API Keys** section and click on the **New API Key** button.
+
+        ![integrations octopus deploy
+        new api key](/images/integrations/octopus-deploy/integrations-octopus-deploy-create-api-key.png)
+
+        Fill out the required fields and click on the **Generate New** button.
```

Comment:
I had this thought but then thought that it might not be a good suggestion to make as never expiring access tokens aren't the safest practice; I'll add an info box stating that the integration will fail if an api key expires - I have a ticket to add editing so will mention they will need to update it

Reply:
![CleanShot 2024-11-25 at 12 43 03@2x](https://github.com/user-attachments/assets/a02615a4-9bed-4427-a9ce-9a63f2d6390e)

Reply:
looks good!